packages/docs/src/routes/tutorial/tutorial-content-footer.tsx

```diff
@@ -1,20 +1,37 @@
-import { component$ } from '@builder.io/qwik';
+import { component$, useSignal } from '@builder.io/qwik';
 import { ensureDefaultFiles, type TutorialStore } from './layout';
 
 export const TutorialContentFooter = component$(({ store }: TutorialContentFooterProps) => {
+  let solutionViewState = useSignal(false);
```

Comment:
```suggestion
  let solutionViewSig = useSignal(false);
```
I think it's more readable like this

Reply:
Done